unittests/snapshot_tests.cpp

```diff
@@ -377,6 +375,7 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(test_chain_id_in_snapshot, SNAPSHOT_SUITE, snapsho
    verify_integrity_hash<SNAPSHOT_SUITE>(*chain.control, *snap_chain.control);
 }
 
+/*
```

Comment:
It appear that maybe I was out of sync with develop and that this test was resurrected.  I'll need to verify that.

Reply:
Good catch. That test was reenabled a couple of weeks ago.